### PR TITLE
177 frontend add testing to notifs

### PR DESCRIPTION
**Pull request by mva919**
# What type of PR is this? (Check all that apply)

- [ ] ✨**feature**: Introduces completely new code or new features.
- [ ] 🐛**fix**: Implements changes that fix a bug. Ideally, reference an issue if present.
- [ ] ♻️**refactor**: Includes any code-related change that is neither a fix nor a feature.
- [ ] ✅**build**: Encompasses all changes related to the build of the software, including changes to dependencies or the addition of new ones.
- [ ] ⚡️**test**: Pertains to all changes regarding tests, whether adding new tests or modifying existing ones.
- [ ] 🚰**ci**: Involves all changes related to the configuration of continuous integration, such as GitHub Actions or other CI systems.
- [ ] 📚**docs**: Includes all changes to documentation, such as README files, or any other documentation present in the repository.
- [x] 🗑️**chore**: Captures all changes to the repository that do not fit into the above categories.

# Description

Add testing to dmNotifs and change dmNotifs and nudgeNotifs to make testing easier (Don't kill me taylor 🥺)

# Tests

How was this tested?

- [x] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [ ] Manual tests
- [ ] Tests were NOT needed
- [ ] Other (explain below)

# [Optional] Screenshots

# Documentation

- [ ] Added to README.me
- [ ] Seperate document
- [x] NO documentation needed

Link to external documentation:

# [Optional] Are there any post-deployment tasks we need to perform?

### CODE REVIEW

File: apps/expo/src/components/notif/dmNotifs/dmNotifs.test.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import { render, screen } from '@testing-library/react-native'
+
+import * as api from '~/utils/api'
```
Comment (thienguen) [inline apps/expo/src/components/notif/dmNotifs/dmNotifs.test.tsx:3]: image

path for this has been updated

image

**Review by thienguen — COMMENTED** [review]
image

works , lgtm

**Review by thienguen — APPROVED** [review]

**Review by taylorfinelli — APPROVED** [review]
LGTM (let's get this money)

**Comment by mva919** [comment]
Thanks for updating the imports T 💅

**Pull request merged**